### DISCUSSION TITLE
Do FreeRTOS supports DLT Logging

### DISCUSSION THREAD

**Discussion by puliviswanath** [COVESA/dlt-daemon#622] [Q&A]
Hi,
I need dlt-daemon for freertos, I'm trying with freeRTOS MSVC demo examples and tried creating a task that executes the dlt-daemon, I also tried different method but facing alot of errors

I want to know whether the dlt-daemon support for freeRTOS if yes, guide me through a detailed steps to configure or create project with dlt-daemon for freeRTOS

**Comment by minminlittleshrimp** [comment]
Hello @puliviswanath 
great idea, for now DLT is not fully supported on FreeRTOS.
I am setting up qemu to run freertos on Linux base. I am also working with a Nuclea STM32, I am going to do a pet project porting DLT to FreeRTOS.
Thanks for your concern.

**Reply by puliviswanath** [reply]
Hello @minminlittleshrimp 
I'm still trying but yet i'm getting stuck, Did it worked for you...?

**Reply by minminlittleshrimp** [reply]
Hello @puliviswanath
I can run successfully Arm cortexM3 MPS2 demo (using QEMU), so I will stick to this demo.
Status: Demo done, implementing DLT into Demo
Issue: DLT build fail with arm toolchain, cannot fully compiled, RCA is in progress

**Comment by minminlittleshrimp** [comment]
For tracking on this topic, follow: https://github.com/users/minminlittleshrimp/projects/5
For repo with source code implementation update: https://github.com/minminlittleshrimp/DLT-on-FreeRTOS
